app/views/ProfileView/index.tsx

```diff
@@ -46,9 +48,9 @@ import useVerifyPassword from '../../lib/hooks/useVerifyPassword';
 const MAX_BIO_LENGTH = 260;
 const MAX_NICKNAME_LENGTH = 120;
 const validationSchema = yup.object().shape({
-	name: yup.string().min(1).required(),
-	email: yup.string().email().required(),
-	username: yup.string().min(1).required()
+	name: yup.string().required(I18n.t('Name_required')),
+	email: yup.string().email(I18n.t('Email_must_be_a_valid_email')).required(I18n.t('Email_required')),
+	username: yup.string().required(I18n.t('Username_required'))
```

Comment:
This is going to create a validation schema based on the current i18n and it won't update the value when the user changes the language, since it's outside of the component.